third-party/TARGETS

```diff
@@ -298,7 +298,6 @@ prebuilt_cxx_library(
     name = "libtorch",
     shared_lib = ":libtorch_gen[libtorch]",
     exported_preprocessor_flags = [
-        "-D_GLIBCXX_USE_CXX11_ABI=0",  # `libtorch` is built without CXX11_ABI so any target depends on it need to use the same build config.
```

Comment:
Let's see if this works. @atalman Do you know if we have switch libtorch build to CXX11_ABI yet?  I don't see it in https://github.com/pytorch/pytorch/pull/143423 yet

Reply:
All the tests are green! Good to land? @huydhn